tools/cli-generator/spec.yaml

```diff
@@ -0,0 +1,86 @@
+commands:
```

Comment:
[q] Is the plan to generate a single spec file from OpenAPI in the future? I can see this file getting quite confusing as we continue to add commands.

Another idea could be to have a separate spec per resource (CRUDL) which would be easier to maintain.

Reply:
I don't know the future, you can see in the spec a lot of things can be guessed I'm sure an improvement would be to make this thing less verbose.

I think we should use it for the next few commands and see how it goes.

Reply:
Another thing is maybe we create an interactive mode that add things to the spec, I don't know yet

Reply:
Cool, makes sense. I like the approach of trying in this project and then refining. Lets see how it works out!